synnax/pkg/distribution/channel/channel.go

```diff
@@ -153,6 +153,11 @@ func (k Keys) Unique() Keys { return lo.Uniq(k) }
 // followed by the keys that are absent in k.
 func (k Keys) Difference(other Keys) (Keys, Keys) { return lo.Difference(k, other) }
 
+// OntologyIDs returns the ontology.ID for each key.
+func (k Keys) OntologyIDs() []ontology.ID {
+	return lo.Map(k, func(key Key, _ int) ontology.ID { return ontology.ID{Type: OntologyType, Key: key.String()} })
```

Comment:
Is there a test for this?

Reply:
This function should be hit as part of client tests. We decided that it's really hard to test at the API level what client tests can already test, so I think much of the server straight up has 0 coverage which is why some of these are being flagged by codecov. I think it's  not worth to add a test now only for a function with very predictable behaviours when in the future we're gonna hit it with more systematic tests. What do you think?

Reply:
Understood. I can agree with this as long we make sure the client tests we make are comprehensive with this in mind. Ill resolve this here then.